models/product-tailoring/src/product-tailoring-data/src/product-tailoring-data-draft/fields-config.ts

```diff
@@ -0,0 +1,35 @@
+import { ReferenceRest } from '@commercetools-test-data/commons';
```

Comment:
@ismaelocaramelo I think you need to refresh this branch with the latest changes in `main`.